### PR TITLE
Bugfix persistency warning

### LINKED ISSUE #387: Files in persistency warning
When running the aminer, i get the following warning:
The AMiner persistence directory should not contain any files.

However, only the aminer logs are in this directory, and they are there by default. I assume these files should be whitelisted for the warning.

### PR DESCRIPTION
**Pull request by whotwagner**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [x] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [x] Issues exist for this PR
- [x] I added related issues using the "Fixes #123'-notations
- [x] This Pull-Requests merges into the "development"-branch

Fixes #387

# Submission specific

- [x] This PR introduces breaking changes
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] All new and existing tests passed

# Describe changes:
- Added configuration for log-directory
- Set default-log-directory to /var/lib/aminer/log
- Autocreate log-directory if it is the default log-directory
- Fixed issue with openjdk-version in testing-dockerfile
- Fixed acl-issue with Dockerfile
- Added a simple Testcase for Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,7 +30,10 @@ RUN apt-get update && apt-get install -y \
         python3-scipy \
         python3-kafka \
         python3-cerberus \
-        python3-yaml 
+        python3-yaml \
+        python3-yaml \
```
Comment (landauermax) [inline Dockerfile:34]: Why do we have python3-yaml twice in this list?

File: Jenkinsfile [hunk 2]
```diff
@@ -91,7 +91,12 @@ pipeline {
                       debianbusterimage = true
                     }
                      sh "docker build -f aecid-testsuite/docker/Dockerfile_deb -t aecid/aminer-debian-buster:$JOB_BASE_NAME-$EXECUTOR_NUMBER-$BUILD_ID --build-arg=varbranch=development --build-arg=vardistri=debian:buster ."
-                     sh "docker run --rm aecid/aminer-debian-buster:$JOB_BASE_NAME-$EXECUTOR_NUMBER-$BUILD_ID"
+                     sh "mkdir -p /tmp/simplerun && mkdir /tmp/simplerun/persistency && mkdir /tmp/simplerun/logs"
+                     sh "cp aecid-testsuite//demo/AMiner/access.log /tmp/simplerun/logs/"
```
Comment (landauermax) [inline Jenkinsfile:95]: I think // can be replaced by /

File: source/root/usr/lib/logdata-anomaly-miner/aminer.py [hunk 3]
```diff
@@ -92,10 +91,29 @@ def initialize_loggers(aminer_config, aminer_user, aminer_grp):
...
+    if log_dir == AMinerConfig.DEFAULT_LOG_DIR:
+        try:
+            os.makedirs(log_dir)
+            from pwd import getpwnam
+            from grp import getgrnam
+            os.chown(log_dir,
+                    getpwnam(aminer_user).pw_uid,
+                    getgrnam(aminer_grp).gr_gid)
+        except FileExistsError:
+            pass
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer.py:104]: Please add a short comment that clarifies why ignoring this error is intentional.

**Review by 4cti0nfi9ure — APPROVED** [review]
Looks good!

**Pull request merged**